src/data/rat-partner.json

```diff
@@ -0,0 +1,89 @@
+{
+    "section-main": {
+        "headline": {
+            "title": "Become a rapid test partner!",
+            "textblock": [
+                "Welcome to the Corona-Warn-App-Community!",
+                "Thank you for helping to break infection chains even faster and more efficiently by connecting your rapid testing center to the Corona-Warn-App (CWA). In doing so, you are making an important contribution to fighting the pandemic and returning to more normality.",
+                "In order to let our partners inform themselves about the connection variants, the onboarding process and the technical specifications at any time, we have compiled the most important information on <a href='https://github.com/corona-warn-app/cwa-quicktest-onboarding/wiki' target='_blank' rel='noopener noreferrer' title=''>GitHub</a>.",
+                "A prerequisite for linking your rapid tests to the Corona-Warn-App is that the test is performed under supervision. This means that no results from so-called lay tests may be transmitted. In addition, it must be ensured that your test center is officially registered with the local health authority and meets the associated requirements."
+            ]
+        },
+        "content": {
+            "headline": {
+                "title": "Collaboration on the project",
+                "text": "There are two options available when connecting the rapid test sites to the Corona-Warn-App."
```

Comment:
This does not really make sense to me. Which project? Where do RAT partners collaborate on which project?